### PR TITLE
Single page workflow

### PR DESCRIPTION
**Pull request by BillWagner**
The purpose of this PR is to test a presentation where all the simple workflows are on a single page, and the more detailed guidance is only needed for those contributors that are creating new content. 

The index page serves as a "front door" where casual contributions require very little investment. Only later, when a contributor wants to do more contributions do we require them to learn more of the complexity.

[Internal review link](https://review.docs.microsoft.com/en-us/contribute/index?branch=pr-en-us-5)

### CODE REVIEW

**Comment by BillWagner** [comment]
@Jim-Parker @wadepickett Can you review the outline for this?

**Comment by rothja** [comment]
@BillWagner When I try to click on the entries in the TOC, some of them don't work. I like this idea. I think there needs to be a single tutorial for making the web edits that is separate from any other content. Do we have that for external contributors? Thanks!!

**Comment by BillWagner** [comment]
@rothja Thanks for your initial thoughts. I haven't created the new articles yet. I wanted to get initial feedback on the TOC and the concept first. I there are no objects, the articles will be added to this PR in the next sprint.

**Comment by rothja** [comment]
@BillWagner Got it. That makes sense. So my only feedback would be that maybe it would be better to not use the overview topic for the simple edits tutorial. Before the Create New Content node, you could have "Make simple edits". It could be a single tutorial. Then the overview could link to that article and the other content in an overview type way? Or is that what we already have? Thanks!

**Comment by Jim-Parker** [comment]
@BillWagner your buckets make sense to me. Have you run this by Sarah for IA considerations?

**Comment by sarah-barrett** [comment]
Looks good, @BillWagner

**Comment by MonicaRush** [comment]
@BillWagner Bill -- Hi -- Have you gotten all the reviews you need? Are you ready to #sign-off? Thanks - Monica

**Comment by BillWagner** [comment]
@MonicaRush I've gotten the reviews I need, and I'll be filling in the remaining work this sprint. It will be next week before its ready for a final review.

**Comment by BillWagner** [comment]
This is ready for final review.
@wadepickett 
@Jim-Parker 
@JRAlexander

File: Contribute/TOC.md [hunk 1]
```diff
@@ -1,30 +1,76 @@
-# [Overview and setup](index.md)
-## [How to contribute](how-to-contribute.md)
-## [Git and GitHub essentials](git-github-fundamentals.md)
+# [Overview and browser based tasks](index.md)
```
Comment (scottaddie) [inline Contribute/TOC.md]: Should "browser based" be hyphenated?

File: Contribute/index.md [hunk 2]
```diff
@@ -14,32 +14,54 @@ ms.custom: external-contributor-guide
 
 Welcome to the [docs.microsoft.com](https://docs.microsoft.com) (Docs) Contributor Guide!
 
-## About this guide
+Several of our documentation sets are Open Source, hosted on GitHub. More teams are adopting this model all the time. It streamlines and improves communication between the product  engineers, the content teams, and our customers. Working in the open provides several advantages:
```
Comment (scottaddie) [inline Contribute/index.md]: There's an extra space between "product" and "engineers".

File: Contribute/index.md [hunk 3]
```diff
@@ -14,32 +14,54 @@ ms.custom: external-contributor-guide
...
+
+You can read new topics before they are published by checking the currently open PRs. Reviews follow the [GitHub flow](https://guides.github.com/introduction/flow/) process. You can see proposed updates or new articles in public repositories. Review them and add your comments. Look at any of our docs repositories, and check the open pull requests (PRs) for areas that interest you. Community feedback on proposed updates helps the entire community.
+
+## Create quality issues
+
+Our docs are a continuous work in progress. Good issues help us focus our efforts on the highest priorities for the community. The more detail you can provide, the more helpful the issue. Tell us what information you sought. Tell us the search terms you used. If you can't get started, tell us how you want to start exploring unfamiliar technology.
+
+Issues start the conversation about what's needed. The content team will respond to these issues with ideas for what we can add, and ask for your opinions. When we create a draft, we'll ask you to [review the PR](#review-open-prs).
+
+## Getting more involved
```
Comment (scottaddie) [inline Contribute/index.md]: Getting more involved --> Get more involved

File: Contribute/index.md [hunk 4]
```diff
@@ -14,32 +14,54 @@ ms.custom: external-contributor-guide
 
 Welcome to the [docs.microsoft.com](https://docs.microsoft.com) (Docs) Contributor Guide!
 
-## About this guide
+Several of our documentation sets are Open Source, hosted on GitHub. More teams are adopting this model all the time. It streamlines and improves communication between the product  engineers, the content teams, and our customers. Working in the open provides several advantages:
 
-Here you'll find all of the information you need to contribute to Docs articles, by using the Microsoft Open Publishing Services (OPS) platform and the supporting tools and processes.
+- We plan in the open to get feedback on what docs are most needed.
```
[the remark this answers is not in the document]
Comment (mikeblome) [reply Contribute/index.md]: Not for C++ at this time. Maybe say "open source repos plan in the open..."

File: Contribute/index.md (hunk 2 above)
Comment (mikeblome) [reply Contribute/index.md]: Can we add a sentence that says something like "Even document sets that are not completely open source have public-facing repos where you are invited to make pull requests."

File: Contribute/index.md [hunk 5]
```diff
@@ -14,32 +14,54 @@ ms.custom: external-contributor-guide
...
+## Quick edits to existing documents
 
-- You can [create issues](how-to-contribute.md#create-issues) to recommend new articles, or improve existing articles.
-- You can [quickly edit](how-to-contribute.md#quick-edits) articles to make small changes in the GitHub online editor.
-- You can [review drafts of new articles](how-to-contribute.md#review-new-articles) to ensure quality and technical accuracy.
-- You can [create new articles](how-to-contribute.md#create-new-articles) for topics when you want to help drive the content story.
-- You can [update](how-to-contribute.md#update-samples) or [create](how-to-contribute.md#create-samples) samples to improve the code samples that reinforce important concepts.
+Quick edits streamline the process to report and fix small errors and omissions in documents. Despite all efforts, small grammar and spelling errors do make their way into our published documents. While you can create issues to report mistakes, you can save both yourself and us time by creating a pull request (PR) to fix the issue. Every article displays an edit button as shown in the following figure. Clicking the edit button takes you to the source on GitHub.
```
[the remark this answers is not in the document]
Comment (JRAlexander) [reply Contribute/index.md]: Many articles?

File: Contribute/index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (BillWagner) [reply Contribute/index.md]: I adopted @mikeblome 's suggestion.

File: Contribute/index.md [hunk 6]
```diff
@@ -14,32 +14,54 @@ ms.custom: external-contributor-guide
...
 
-## Contribution tasks
+> [!IMPORTANT]
+> All repositories that publish to docs.microsoft.com have adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/) or the [.NET Foundation Code of Conduct](https://dotnetfoundation.org/code-of-conduct). For more information, see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/). Or contact [opencode@microsoft.com](mailto:opencode@microsoft.com), or [conduct@dotnetfoundation.org](mailto:conduct@dotnetfoundation.org) with any questions or comments.<br>
+>
+> Minor corrections or clarifications to documentation and code examples in public repositories are covered by the [docs.microsoft.com Terms of Use](https://docs.microsoft.com/legal/termsofuse). New or significant changes generate a comment in the pull request, asking you to submit an online Contribution License Agreement (CLA) if you are not an employee of Microsoft. We need you to complete the online form before we can accept your pull request.
```
Comment (Rick-Anderson) [inline Contribute/index.md]: The online CLA form must be completed before we can review your PR.

We shouldn't be reviewing a PR without a signed agreement.

**Pull request merged**